lua/gh-actions/github.lua

```diff
@@ -5,7 +5,7 @@ local utils = require('gh-actions.utils')
 local M = {}
 
 ---@param str string
----@return string
+---@return table<integer, string>
```

Comment:
This function still returns a string :)

```suggestion
---@return string
```